css-positioning/absolute.css

```diff
@@ -0,0 +1,41 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  top: 40px;
+  left: 230px;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: relative;
```

Comment:
if you set `position` on the `.meme` selector, then this could be set to `position: absolute` -- `.joke` will be positioned against `.meme` when the `.meme` is a non-static element